docs/src/AvatarGroup.doc.js

```diff
@@ -111,6 +111,29 @@ See the [Accessibility guidelines](#Accessibility) for details on proper usage.`
   />,
 );
 
+card(
+  <MainSection name="Usage guidelines">
+    <MainSection.Subsection columns={2}>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        title="When to Use"
+        description={`
+          - For the general display of groups of people, companies and/or brands.
```

Comment:
probably worth mentioning that this can also be used as a button to add more collaborators, which is unique to AvatarGroup

Reply:
Good call - I'll add!